### DISCUSSION TITLE
What is the setting in NVDA to turn off a flood of "Selection area changed dimensions" notifications?

### DISCUSSION THREAD

**Discussion by ChrisKerIntel** [nvaccess/nvda#16389] [Q&A]
When I am using NVDA and trying to take a screenshot of something, at the point where I am using the mouse to drag to select an area of the screen for a screenshot, I am flooded with "Selection area changed dimensions" notifications reflecting the position of the mouse pointer. I haven't found a setting that turns this off, in braille, mouse or other settings. 

How do I turn this off? 

This greatly inhibits my ability to take screenshots of issues with the speech viewer on, because the "Selection area changed dimensions" notifications overrun the speech viewer and prevent me from including meaningful content of it in screenshots.

**Comment by CyrilleB79** [comment]
Hello

I cannot reproduce this issue on my Windows version.
But what about opening Object presentation settings (NVDA+control+O) and unchecking "Report notifications"?

**Comment by ChrisKerIntel** [comment]
Hi – neither unchecking that, nor unchecking anything else in Object Notifications, fixed it…

**Comment by ChrisKerIntel** [comment]
Still reproducing this with latest NVDA, no modifications to default settings... I'm finding that the stream of notifications isn't showing up in the screenshot, thankfully, but it's making it a bit trial-and-error still to determine whether I'm including the right info in the screenshot re: NVDA Speech Viewer window's content.

**Reply by hwf1324** [reply]
The problem is caused by Windows screenshots, which may be prompted by the software.
You may be able to bypass this problem by switching to another screenshot program.

**Comment by AzeemQ1596** [comment]
I'm having this issue as well. It wasn't happening since last week. Before it was working fine. Not sure what changed. I tried resetting my NVDA settings but nothing no fix. Was there some update to how Windows 11 screenshot works?

**Comment by CyrilleB79** [comment]
@ChrisKerIntel, @AzeemQ1596, I can now reproduce here; probably I have nad a windows update in the meantime.

The issue is due to UIA notifications provided by Windows or the application itself. NVDA cannot natively remove them. By the way, Narrator is flooded the same way in its speech recap window.

Fortunately, there is an add-on allowing to silence these notifications! It's called UIANotificationSwitch; the author is Shunxian Wu (@w-sx).
Just install it and press NVDA+I to set UIA notification on beep or on off.

Hope this solves your problem!
